### DISCUSSION TITLE
Multiple Cursors in Single User mode

### DISCUSSION THREAD

**Discussion by ashu12chi** [ueberdosis/tiptap#1441] [General Questions]
Hello,

I want to implement multiple cursors in single user mode to achieve what many popular editors provide like VS Code. I found one [issue](https://github.com/ueberdosis/tiptap/issues/1310) similar to this. Here, it is mentioned that this functionality won't be provided. I want to implement this on my own. Can anyone provide some guidance, how can I approach to solve this?

**Comment by philippkuehn** [comment]
This is not possible with a "real" cursor. What you can try is to simulate cursors with decorations and redo all transactions for all simulated cursor positions. But that will be a huge amount of work.
